app/controllers/items/new.js

```diff
@@ -177,6 +189,30 @@ export default GoodcityController.extend(
       return detailAttributes;
     },
 
+    clearParams() {
+      if (this.get("displayFields")) {
+        let attr = this.get("paramsNotCopied");
+        let fieldAttributes = this.get("displayFields").map(
+          value => value.name
+        );
+        attr.forEach(value => {
```

Comment:
Just a suggestion.. move this iteration logic to other function to make it more readable like

```suggestion
        attr.forEach(this.clearAttribute)
```